slrealizer/make_nonlens_source_object.py

```diff
@@ -0,0 +1,29 @@
+import os, sys
```

Comment:
Same comment here about documenting scripts. All code needs internal documentation - your future self will thank you for it ;-)

Reply:
Combined the scripts for lenses and non-lenses into a demo notebook, at `demo/Example+SLRealizer+Usage.ipynb`